pkg/cache/placeholder_test.go

```diff
@@ -33,26 +33,71 @@ import (
 	siCommon "github.com/apache/yunikorn-scheduler-interface/lib/go/common"
 )
 
+const (
+	gpu       = "nvidia.com/gpu"
+	ephemeral = "ephemeral-storage"
+	hugepages = "hugepages-1Gi"
```

Comment:
Nit: these can referenced from core/v1: `v1.ResourceEphemeralStorage` / `v1.ResourceHugePagesPrefix`

Reply:
BTW, "hugepages-xxx" can have multiple values, do we only care about `1Gi`?

Reply:
No, in the check we only look at the prefix `hugepages-` anything with that prefix is filtered out